src/osd/OSD.cc

```diff
@@ -3712,14 +3717,14 @@ void OSD::handle_pg_peering_evt(
 {
   if (service.splitting(pgid)) {
     peering_wait_for_split[pgid].push_back(evt);
-    return;
+    return -EEXIST;
```

Comment:
speaking of which, we should probably stop sending create messages for split pgs entirely.. we already ignore them!